content/docs/authorization/cli.md

```diff
@@ -726,6 +724,7 @@ karavictl storage create [flags]
 ##### Optional Flags
 ```
   -h, --help       Help for delete
+  -a, --array-insecure   Skip certificate validation of the storage array
```

Comment:
Can you align the flags with these changes?

Reply:
Fixed.